detection_rules/config.py

```diff
@@ -158,13 +205,17 @@ def parse_rules_config(path: Optional[Path] = None) -> RulesConfig:
         loaded = yaml.safe_load(path.read_text())
     elif CUSTOM_RULES_DIR:
         path = Path(CUSTOM_RULES_DIR) / '_config.yaml'
-        assert path.exists(), f'_config.yaml file missing in {CUSTOM_RULES_DIR}'
         loaded = yaml.safe_load(path.read_text())
     else:
         path = Path(get_etc_path('_config.yaml'))
         loaded = load_etc_dump('_config.yaml')
 
-    assert loaded, f'No data loaded from {path}'
+    try:
+        ConfigFile.from_dict(loaded)
+    except KeyError as e:
+        raise SystemExit(f'Missing key `{str(e)}` in _config.yaml file.')
+    except TypeError:
+        raise SystemExit(f'No data loaded from {path}')
```

Comment:
```suggestion
        raise SystemExit(f'No data loaded from {path}')
    except ValueError as e:
        raise SystemExit(e)
```

Reply:
We probably don't need this. Might be good to add just in case. If this is related to `RulesConfig`, remember the initialization happens later.